src/shapes/Circle.ts

```diff
@@ -19,20 +19,25 @@ interface UniqueCircleProps {
   radius: number;
 
   /**
-   * degrees of start of the circle.
-   * probably will change to degrees in next major version
-   * @type Number 0 - 359
+   * Angle for the start of the circle, in degrees.
+   * @type TDegree 0 - 359
    * @default 0
    */
   startAngle: number;
 
   /**
-   * End angle of the circle
-   * probably will change to degrees in next major version
-   * @type Number 1 - 360
+   * Angle for the end of the circle, in degrees
+   * @type TDegree 1 - 360
    * @default 360
    */
   endAngle: number;
+
+  /**
+   * Orientation for the direction of the circle.
+   * Setting to true will switch the arc of the circle to traverse from startAngle to endAngle in a counterclockwise direction,
+   * @default false
+   */
```

Comment:
Can we be extremely safe and explain with an extra line that this counterClockwise is not going to affect anything else like the rotate method or the normal angle of the circle? I don't want devs to find this and then being confused that the circle doesn't spin counter clockwise.